packages/flutter/lib/src/widgets/drag_target.dart

```diff
@@ -64,7 +64,7 @@ typedef DragTargetLeave = void Function(Object? data);
 /// Signature for when a [Draggable] moves within a [DragTarget].
 ///
 /// Used by [DragTarget.onMove].
-typedef DragTargetMove = void Function(DragTargetDetails<dynamic> details);
+typedef DragTargetMove<T> = void Function(DragTargetDetails<T> details);
```

Comment:
I think this can also be applied to DragTargetLeave

Reply:
Oh yes! Done.